exasol_transformers_extension/udfs/question_answering_udf.py

```diff
@@ -112,7 +121,8 @@ def _predict_model(self, model_df: pd.DataFrame) -> \
         question_answerer = self.pipeline(
```

Comment:
question, shouldn't we create the pipeline outside the prediction loop in load_models.